test/pgtest/vars.pt

```diff
@@ -0,0 +1,72 @@
+# Test status codes for errors related to setting session variables.
+
+# FixedValueParameter and InvalidParameterValue are not covered here
```

Comment:
We have a `pgtest-mz` directory for non-postgres compatible stuff if you'd like to add these.

Reply:
Ah, nice, will do!